src/main/no/nav/pto/veilarbfilter/HttpHandler.kt

```diff
@@ -71,7 +71,7 @@ fun createHttpServer(
                 readinessCheck = { applicationState.initialized },
                 livenessCheck = { applicationState.running })
             route("/api/") {
-                veilederGruppeRoutes(veilederGrupperService, PepClient(config = configuration))
+                veilederGruppeRoutes(veilederGrupperService, PepClient(config = configuration),useAuthentication)
```

Comment:
I suppose we can pass this parameter `useAuthentication` in configuration, if its something global, if its not then ignore my comment.

Reply:
my mistake...ignore this comment, now I see its good how you did it..sorry